kong/utils_test.go

```diff
@@ -159,3 +161,356 @@ func Test_requestWithHeaders(t *testing.T) {
 		})
 	}
 }
+
+func TestFillRoutesDefaults(T *testing.T) {
+	assert := assert.New(T)
+
+	client, err := NewTestClient(nil, nil)
+	assert.Nil(err)
+	assert.NotNil(client)
+
+	tests := []struct {
+		name     string
+		route    *Route
+		expected *Route
+	}{
+		{
+			name: "name paths",
```

Comment:
```suggestion
			name: "fills defaults for all fields except paths, leaves name as is",
```
Descriptive tests, please.